kubernetes-dns-node-cache-1.22.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: kubernetes-dns-node-cache-1.22
+  version: 1.22.20
+  epoch: 0
+  description: NodeLocal DNSCache improves Cluster DNS performance by running a DNS caching agent on cluster nodes as a DaemonSet.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - build-base
+      - ca-certificates-bundle
+      - busybox
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes/dns
+      tag: ${{package.version}}
+      expected-commit: 7bcdf8411514fd4e22de11e93f33559c7304ed21
+
+  - runs: |
+      # Mitigate GHSA-xc8m-28vv-4pjc, GHSA-cgcv-5272-97pr, GHSA-qc2g-gmh6-95p4
+      go get k8s.io/kubernetes@v1.24.15
+
+      go mod tidy
+      go mod vendor
+
+  - uses: go/build
+    with:
+      packages: ./cmd/node-cache
+      output: node-cache
+      ldflags: -s -w -X github.com/kubernetes/dns/pkg/version.Version=v${{package.version}}
+
+  - uses: strip
+
+subpackages:
+  - name: kubernetes-csi-node-driver-registrar-compat
```

Comment:
should we add version information here as well for compat package?

Reply:
also it seems that it should be `kubernetes-dns-node-cache-1.22-compat`, not `kubernetes-csi-node-driver-registrar-compat`